cameo/strain_design/heuristic/evolutionary/objective_functions.py

```diff
@@ -208,15 +208,13 @@ def __call__(self, model, solution, targets):
                 else:
                     substrate_flux += abs(solution.fluxes[substrate_id]) * n_carbon(substrate) / 2
             substrate_flux = round(substrate_flux, config.ndecimals)
-
         else:
             product_flux = round(solution.fluxes[self.product], config.ndecimals)
             substrate_flux = round(sum(abs(solution.fluxes[s]) for s in self.substrates), config.ndecimals)
-        try:
+        if substrate_flux > config.non_zero_flux_threshold:
             return (biomass_flux * product_flux) / substrate_flux
-
-        except ZeroDivisionError:
-            return 0.0
+        else:
+            return 0.
```

Comment:
Alright, this PR was about getting rid of the ZeroDivError so yep, let's handle the improvement in a different pr